.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+<!--- Please include the phrase "BREAKING CHANGE:" here if your require a major release -->
+
+<!--- Don't forget to note any issues here with "fixes #<issue number>" -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- For bug fixes, include regression unit tests that fail without the fix -->
+<!--- For new features, include unit tests for the new functionality -->
+
+## Screenshots (if appropriate):
+
+## Checklist:
+<!--- Please mark an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have read the **CONTRIBUTING** document.
```

Comment:
I wonder if you can link to the `CONTRIBUTING.md` file? Mainly for other folks looking at the PR

Reply:
does it need to be a relative path since this file is nested in .github/ ? is there a way to test these readmes out before merging?